apps/system/tzsh/Kconfig

```diff
@@ -0,0 +1,91 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config SYSTEM_TZSH
```

Comment:
TizenRT already supported shell(TASH). Why did you add new shell?

Reply:
It is more powerful than you have.